source/config.hpp

```diff
@@ -79,6 +80,9 @@ class Config
 	string const &default_member_rvalue_reference_return_value_policy() { return default_member_rvalue_reference_return_value_policy_; }
 	string const &default_call_guard() { return default_call_guard_; }
 
+	string const &custom_shared() const { return custom_shared_; }
+	bool use_custom_shared() const { return !custom_shared_.empty(); }
```

Comment:
Do we really need this flag?

Reply:
No, we don't if we do as discussed above.